docs/site/downloads/cldr-48.md

```diff
@@ -76,7 +74,7 @@ For a full listing, see [Coverage Levels](https://unicode.org/cldr/charts/dev/su
 
 ## Specification Changes
 
-The following are the most significant changes to the specification (LDML).
+The following are some of the most significant changes to the specification (LDML).
```

Comment:
I thought the modifications is a list of the significant changes? Where would any significant changes be that aren't mentioned in the release page?

Maybe if we think there are things in the modifications of the spec but not on the release page we should say something along the lines of 'The following are some of the most significant changes to the specification (LDML). The [modifications] section contains a list of all substantiative changes.' or something similar?

Reply:
> it up front

Not sure what the "it" is.

Reply:
I think this sounds subjective. I would prefer then "The following are some of the changes..." if we want to hedge.

Reply:
See above comment.